src/pages/_id/index.vue

```diff
@@ -201,6 +209,14 @@ export default {
         }
       }
     },
+    async loginAddress() {
+      if (!this.getFollowers.length && this.loginAddress) {
+        if (!this.walletHasLoggedIn) {
+          await this.signLogin();
```

Comment:
Doesn't `loginAddress` changes only if the new user has logged in though, or are we trying to handle `loginAddress === ''` here?